features/enlinkd/tests/src/test/java/org/opennms/netmgt/enlinkd/Nms17216EnIT.java

```diff
@@ -745,10 +757,146 @@ public void testNetwork17216CdpTopology() throws Exception {
 
         assertTrue(m_linkd.runSingleSnmpCollection(switch1.getId()));
         assertEquals(5, m_cdpLinkDao.countAll());
+        assertTrue(m_cdpTopologyService.hasUpdates());
         
         assertTrue(m_linkd.runSingleSnmpCollection(switch2.getId()));
         assertEquals(11, m_cdpLinkDao.countAll());
+        assertTrue(m_cdpTopologyService.hasUpdates());
+        
+        // Now we support by default configuration: BRIDGE, CDP, ISIS, LLDP and OSPF
+        assertEquals(5, m_topologyDao.getSupportedProtocols().size());
```

Comment:
This is out of place - why are we asserting that the set of supported protocols in the topologyDao has these 5 protocols in a test called `testNetwork17216CdpTopology` in the `Nms17216EnIT.java`?